site/_includes/partials/devtools/pt/whats-new.md

```diff
@@ -1,2 +1,14 @@
 ## More DevTools features {: #whats-new }
```

Comment:
Please help to translate all the files in `site/_includes/partials/devtools/pt/*` to English.

Reply:
Done!